### ISSUE TITLE
API Metadata issue ;(  Please update to api48 and add lacking fields

### ISSUE THREAD

**Issue by donpablom** [scolladon/sfdc-generate-package#36]
Due to generating manifest, if some metadata exist in api and don't exist in soap api it not appear in  metadata.js file. Hope you still care about this tool ;) 

Lacking metadata - [permissionsetgroups](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_permissionsetgroup.htm)

After adding it it to file, works like charm, please add this metadata.
``` 
  "permissionsetgroups": {
    "xmlName": "PermissionSetGroup",
    "minApiVersion": 45.0,
    "maxApiVersion": 47.0
  },
```

P.S. **and update to API 48**

**Comment by enriquedurango** [comment]
good point I was testing today with the api version 48 and it does not work.

**Comment by scolladon** [comment]
Hi,

Thanks for your interest in the tool guys
Could you create a pull request with this metadata ?

Seb
